projects/golang/build.sh

```diff
@@ -0,0 +1,58 @@
+# Extract golang, set paths
+tar zxvf go1.11.1.linux-amd64.tar.gz
+export GOROOT=`realpath go`
+export GOPATH=$GOROOT/packages
+export PATH=$GOROOT/bin:$PATH
+
+# go-fuzz-build is the tool that instruments Go files
+go build github.com/guidovranken/libfuzzer-go/go-fuzz-build
+
+function compile_fuzzer {
+    fuzzer=$(basename $1)
+
+    # main.c is the shim between the instrumented Go code generated by go-fuzz-build and libFuzzer
+    $CC $CFLAGS -Wall -Wextra $GOPATH/src/github.com/guidovranken/libfuzzer-go/C/main_libFuzzer_extra_counters.c -g -O3 -c -o main.o
+
+    # Instrument all Go files relevant to this fuzzer, compile and store in $fuzzer.a
+    ./go-fuzz-build -o $fuzzer.a github.com/dvyukov/go-fuzz-corpus/$fuzzer
+
+    # Instrumented, compiled Go ($fuzzer.a) + shim (main.o) + libFuzzer = fuzzer binary
+    $CXX $CXXFLAGS main.o $fuzzer.a -lFuzzingEngine -lpthread -o fuzzer-$fuzzer
```

Comment:
Good news everyone!
@guidovranken change for libfuzzer support is merged into go-fuzz.
We now also don't need the C shim.